examples/microcoap_server/README.md

```diff
@@ -4,25 +4,32 @@ microcoap server example
 This application is meant to get you started with impementing a CoAP server on RIOT.
 It uses the gnrc network stack through RIOT's conn socket API.
 
-
 Usage
 =====
 
-To try on native, compile with
+To try out the server on native, compile it with
+
+```
+$ make all
+```
+
+Then, create a tap interface (to which RIOT will connect) and a bridge (to which
```

Comment:
>Argh, that happens when you just copy&paste.. >.< but then should https://github.com/RIOT-OS/RIOT/tree/master/examples/gnrc_networking be corrected as well?

The bridge is needed to let two native nodes communicate (for `gnrc_networking`).  It would make sense to talk about this in the README (of `gnrc_networking`) if you are going to touch it anyways (: